packages/artalk/src/components/html/list.html

```diff
@@ -1,13 +1,10 @@
 <div class="atk-list">
   <div class="atk-list-header">
     <div class="atk-comment-count">
-      <div class="atk-text">
-        <span class="atk-comment-count-num">0</span>
-        条评论
-      </div>
+      <div class="atk-text"></div>
     </div>
     <div class="atk-right-action">
-      <span data-action="admin-close-comment" class="atk-hide" atk-only-admin-show>关闭评论</span>
+      <span data-action="admin-close-comment" class="atk-hide" atk-only-admin-show></span>
```

Comment:
Why has the text been removed here?

Reply:
Yep, you're right. No need to be removed. DOM action will overwrite this, but keep it may more clear on code reading